docs/gettingstarted/manifest-deployer-tutorial/README.md

```diff
@@ -0,0 +1,358 @@
+  - [Preparation of work environment](#preparation-of-work-environment)
+    - [Structure of demo material](#structure-of-demo-material)
+    - [Install the Landscaper together with an OCI registry](#install-the-landscaper-together-with-an-oci-registry)
+  - [Deploy applications via Landscaper](#deploy-applications-via-landscaper)
+    - [1. Push demo image content into OCI registry](#1-push-demo-image-content-into-oci-registry)
+    - [2. Develop Landscaper artifacts](#2-develop-landscaper-artifacts)
+    - [3. Push Component Archive to OCI Registry](#3-push-component-archive-to-oci-registry)
+    - [4. Deploy the application](#4-deploy-the-application)
+
+## Introduction
+
+This tutorial describes how to use the Landscaper manifest deployer for installing an application into a Kubernetes cluster.
+
+The demo is ordered into the following activities:
+
+  - [1. Push demo image content into OCI registry](#1-push-demo-image-content-into-oci-registry)
+  - [2. Develop Landscaper artifacts](#2-develop-landscaper-artifacts)
+  - [3. Push Component Archive to OCI Registry](#3-push-component-archive-to-oci-registry)
+  - [4. Deploy the application](#4-deploy-the-application)
+
+![alt text](README-deployment-4-steps.svg "Deployment with Landscaper in four steps")
+
+## Preparation of work environment
+
+Before you start, you need a working demo environment. If you do not have one, please refer to ["Prepare demo environment"](prepare-demo-environment.md).
+
+First, download [demo material from this repository]([/](https://download-directory.github.io/?url=https%3A%2F%2Fgithub.com%2Fgardener%2Flandscaper%2Ftree%2Fmaster%2Fdocs%2Fgettingstarted%2fmanifest-deployer-tutorial)). You will edit some of the downloaded files later.
+
+### Structure of demo material
+
+The folder demo content contains a sample OCI image.
+The folder manifests contains all Kubernetes manifests.
+The folder component-archive contains all resources needed for building Landscaper deployable component.
+
+``` text
+./manifest-deployer-tutorial/
+├── component-archive
+│   ├── blueprint
+│   │   ├── blueprint.yaml
+│   │   └── deploy-execution.yaml
+│   ├── component-descriptor.yaml
+│   └── resources.yaml
+├── demo-content
+│   └── hello.tar
+├── manifests
+│   ├── installation.yaml
+│   ├── my-deployment.yaml
+│   ├── my-secret.yaml
+│   └── my-target.yaml
+├── prepare-demo-environment.md
+└── README.md
+```
+
+| Demo environment | Productive environment |
+| ------------- |-------------|
+| For demo purposes we will use a simplified setup with one cluster, which contains all necessary parts. | In a productive environment could be separated. |
+| ![alt text](README-env-demo.svg "demo all-in-one cluster") | ![alt text](README-env-productice.svg "One demo cluster and other possible setup")|
+
+### Install the Landscaper together with an OCI registry
+
+The landscaper-cli provides a convenient quick start option for installing the Landscaper plus an OCI registry:
```

Comment:
This is very confusing? Why do we install an OCI registry here? This comes without any motivation.